microdata_validator/temporal_attributes.py

```diff
@@ -0,0 +1,177 @@
+            "description": [
+                {
+                    "languageCode": "no",
+                    "value": "Startdato/måletidspunktet for hendelsen",
+                },
+                {
+                    "languageCode": "en",
+                    "value": "Startdato/måletidspunktet for hendelsen",
+                },
+            ],
+            "name": [
+                {"languageCode": "no", "value": "Startdato"},
+                {"languageCode": "en", "value": "Start date"},
+            ],
+        },
+        "STOP": {
+            "description": [
+                {
+                    "languageCode": "no",
+                    "value": "Startdato/måletidspunktet for hendelsen",
+                },
+                {
+                    "languageCode": "en",
+                    "value": "Startdato/måletidspunktet for hendelsen",
+                },
+            ],
+            "name": [
+                {"languageCode": "no", "value": "Stoppdato"},
+                {"languageCode": "en", "value": "Stop date"},
+            ],
+        },
+    },
+    "EVENT": {
+        "START": {
+            "description": [
+                {
+                    "languageCode": "no",
+                    "value": "Startdato/måletidspunktet for hendelsen",
+                },
+                {
+                    "languageCode": "en",
+                    "value": "Startdato/måletidspunktet for hendelsen",
+                },
+            ],
+            "name": [
+                {"languageCode": "no", "value": "Startdato"},
+                {"languageCode": "en", "value": "Start date"},
+            ],
+        },
+        "STOP": {
+            "description": [
+                {
+                    "languageCode": "no",
+                    "value": "Startdato/måletidspunktet for hendelsen",
+                },
+                {
+                    "languageCode": "en",
+                    "value": "Startdato/måletidspunktet for hendelsen",
+                },
+            ],
+            "name": [
+                {"languageCode": "no", "value": "Stoppdato"},
+                {"languageCode": "en", "value": "Stop date"},
+            ],
+        },
+    },
+}
+
+START_VARIABLE_DEFINITION = {
+    "variableRole": "START_TIME",
+    "shortName": "START",
+    "dataType": "DATE",
+    "valueDomain": {
+        "description": [
+            {
+                "languageCode": "no",
+                "value": "Dato oppgitt i dager siden 1970-01-01"
+            }
+        ]
+    },
+}
+STOP_VARIABLE_DEFINITION = {
+    "variableRole": "STOP_TIME",
+    "shortName": "STOP",
+    "dataType": "DATE",
+    "valueDomain": {
+        "description": [
+            {
+                "languageCode": "no",
+                "value": "Dato oppgitt i dager siden 1970-01-01"
+            }
+        ]
+    },
+}
+
+
+def generate_start_time_attribute(temporality_type: str):
+    start_attribute = START_VARIABLE_DEFINITION.copy()
```

Comment:
Wouldn't deepcopy() be more correct here since the dict contains other objects (lists)? You always update start_attribute so it works correctly with the current code.